ir-black-theme.el

```diff
@@ -55,8 +56,10 @@
       (*light-purple*       "#FFCCFF")
       (*line-number*        "#3D3D3D")
       (*method-declaration* "#FFD2A7")
-      (*mode-line-bg*       "#202020")
-      (*mode-line-fg*       "#CCCCCC")
+      (*mode-line-bg*       "grey70")
+      (*mode-line-fg*       "black")
+      (*mode-line-inactive* "grey30")
```

Comment:
Would you be able to convert these to hex for consistency with the other definitions? http://www.farb-tabelle.de/en/rgb2hex.htm?q=gray30 seems like it'll do the trick.